includes/functions_import.php

```diff
@@ -1,5 +1,457 @@
 <?php
 
```

Comment:
You'll want to check off line, but as I said, the logic is copy pasted from original locations. Only changed the input parameters and the return for the second function.